### ISSUE TITLE
.groupSpacing for multiBarChart should be available for multiBarHorizontalChart

### ISSUE THREAD

**Issue by HopefulLlama** [novus/nvd3#1058]
The .groupSpacing(x) function is useful for charts with a lot of data points as it allows you to define larger spacing between the groups of bars.

For some reason though, this function isn't available for the horizontal bar chart.

**Comment by robinfhu** [comment]
Yep, you're right.   Don't know why that is though.  Perhaps you can open a PR?

**Comment by slambdin1** [comment]
+1
